lib/rautomation/platform.rb

```diff
@@ -1,8 +1,15 @@
 module Platform
    extend self
 
-   def architecture
-     is_x86? ? 'x86' : 'x64'
+   def architecture(arg)
+      case arg.include?('x86')
+      when true
+         'x86'
+      when false
+         'x64'
+      else
+         raise ArgumentError, "Unable to resolve architecture based on arg : #{arg}"
```

Comment:
This case can never happen since `include?` always returns a boolean. Just change it to the code I mentioned in the other comment or use regular `if-else`.